include/exadg/incompressible_navier_stokes/spatial_discretization/operator_pressure_correction.cpp

```diff
@@ -328,6 +330,34 @@ OperatorPressureCorrection<dim, Number>::evaluate_nonlinear_residual(
   dst.add(-1.0, *rhs_vector);
 }
 
+template<int dim, typename Number>
```

Comment:
rhs for Picard in the pressure correction scheme, similar to Stokes rhs once again.